utils/appliance.py

```diff
@@ -1628,6 +1628,57 @@ def reset_automate_model(self):
         with self.ssh_client as ssh_client:
             ssh_client.run_rake_command("evm:automate:reset")
 
+    def configure_appliance_for_openldap_ext_auth(self, appliance_name):
+        """Sets up the appliance for an external authentication with OpenLdap.
+
+        Keywords:
+            get_groups: Get User Groups from External Authentication (httpd).
+            ipaserver: IPA server address.
+            iparealm: Realm.
+            credentials: Key of the credential in credentials.yaml
+        """
+        openldap_domain1 = conf.cfme_data['auth_modes']['ext_openldap']
+        appliance_fqdn = '{}.{}'.format(appliance_name, openldap_domain1['domain_name'])
+        # updating the /etc/hosts is a workaround due to the
+        # https://bugzilla.redhat.com/show_bug.cgi?id=1360928
+        rc, out = self.ssh_client.run_command('appliance_console_cli --host {}'.format(
+            appliance_fqdn))
+        assert rc == 0, out
```

Comment:
it is now possible to assert the ssh result directly, so you can do it like this:
```python
assert self.ssh_client.run_command('appliance_console_cli --host {}'.format(appliance_fqdn))
```

In this case the result acts like a bool.